src/pilot/templates/nic-configs/4_port/compute-ovs-dpdk-mode2.yaml

```diff
@@ -0,0 +1,238 @@
+    type: number
+  ControlPlaneSubnetCidr: # Override this via parameter_defaults
+    default: '24'
+    description: The subnet CIDR of the control plane network.
+    type: string
+  ControlPlaneDefaultRoute: # Override this via parameter_defaults
+    description: The default route of the control plane network.
+    type: string
+  ExternalInterfaceDefaultRoute: # Not used by default in this template
+    default: '10.0.0.1'
+    description: The default route of the external network.
+    type: string
+  ManagementInterfaceDefaultRoute: # Commented out by default in this template
+    default: unset
+    description: The default route of the management network.
+    type: string
+  DnsServers: # Override this via parameter_defaults
+    default: []
+    description: A list of DNS servers (2 max for some implementations) that will be added to resolv.conf.
+    type: comma_delimited_list
+  EC2MetadataIp: # Override this via parameter_defaults
+    description: The IP address of the EC2 metadata server.
+    type: string
+  ComputeBond0Interface1:
+    default: 'em1'
+    description: Bond 0 interface 1 name
+    type: string
+  ComputeBond0Interface2:
+    default: 'p3p1'
+    description: Bond 0 interface 2 name
+    type: string
+  ComputeBond1Interface1:
+    default: 'em2'
+    description: Bond 1 interface 1 name
+    type: string
+  ComputeBond1Interface2:
+    default: 'p3p2'
+    description: Bond 1 interface 2 name
+    type: string
+  ComputeBondInterfaceOptions:
+    default: '802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1'
+    description: The bonding_options string for the bond interface.
+    type: string
+  BondInterfaceOptions:
+    default: ''
+    description: The bonding_options string for the bond interface. Set things
+                 like mode=802.3ad using this option.
+    type: string
+  OvsDpdkBondInterfaceOptions:
+    default: ''
+    description: The bonding_options string for the bond interface. Set things
+                 like 'bond_mode=balance-tcp lacp=active' using this option.
+    type: string
+
+resources:
+  OsNetConfigImpl:
+    type: OS::Heat::StructuredConfig
+    properties:
+      group: os-apply-config
+      config:
+        os_net_config:
+          network_config:
+            -
+              type: ovs_bridge
+              name: br-tenant
+              use_dhcp: false
+              dns_servers: {get_param: DnsServers}
+              addresses:
+                -
+                  ip_netmask:
+                    list_join:
+                      - '/'
+                      - - {get_param: ControlPlaneIp}
+                        - {get_param: ControlPlaneSubnetCidr}
+              routes:
+                -
+                  ip_netmask: 169.254.169.254/32
+                  next_hop: {get_param: EC2MetadataIp}
+                -
+                  default: true
+                  next_hop: {get_param: ControlPlaneDefaultRoute}
+              
```

Comment:
The above line has several spaces on it.  Can you remove the spaces and leave the blank line?

Reply:
Done.